tests/ui/lint/non-snake-case/lint-non-snake-case-crate.proc_macro_.stderr

```diff
@@ -1,14 +1,12 @@
 error: crate `NonSnakeCase` should have a snake case name
-  --> $DIR/lint-non-snake-case-crate.rs:25:18
+  --> $DIR/lint-non-snake-case-crate.rs:29:18
    |
 LL | #![crate_name = "NonSnakeCase"]
    |                  ^^^^^^^^^^^^ help: convert the identifier to snake case: `non_snake_case`
    |
 note: the lint level is defined here
-  --> $DIR/lint-non-snake-case-crate.rs:27:9
+  --> $DIR/lint-non-snake-case-crate.rs:31:9
    |
 LL | #![deny(non_snake_case)]
    |         ^^^^^^^^^^^^^^
 
-error: aborting due to 1 previous error
```

Comment:
I needed to remove this line because the "compare-output-lines-by-subset" directive only does line-by-line comparison, and this line looks like the following when the compiler omits a warning:

```
error: aborting due to 1 previous error; 1 warning emitted
```

Reply:
Went with using the needs-unwind directive instead.